app/views/account_requests/confirmation.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="card">
+  <div class="card-body text-center">
+    <div class='card-text'>
+      <h3> Are you sure you that you want an account with us? </h3>
```

Comment:
OK, *not a blocker*, but:

I think the language here reads funny, in context. I understand the confirmation pattern you're using here, but maybe we could use more assertive language? Something like:
```suggestion
      <h3> Are you ready to get started? </h3>
```

and then 👇